cf_remote/commands.py

```diff
@@ -358,6 +358,20 @@ def spawn(platform, count, role, group_name, provider=Providers.AWS, region=None
 
     return 0
 
+def _is_saved_group(vms_info, group_name):
+    group = vms_info[group_name]
+    return ("meta" not in group
+            or "region" not in group["meta"]
+            or "provider" not in group["meta"])
```

Comment:
I would actually add an explicit `"saved": true` to `meta` for such groups.

Reply:
Done.